docs/extensibility/extend-rest-api-formatters.md

```diff
@@ -0,0 +1,95 @@
+# Formatters
+As part of ExtendRestAPI, we introduced `Formatters`, these are utility classes that allow you to format values to
+so that they are compatible with the StoreAPI, values such as money, currency, or HTML.
```

Comment:
What do you think about  adding here a list of the formatters as links, to be used as quick jump lists

- MoneyFormatter
- CurrencyFormatter
- HtmlFormatter

Reply:
Done, thanks for the suggestion! :)